witness/src/Dscp/Resource/Network.hs

```diff
@@ -123,9 +126,15 @@ instance WithNetLogging => AllocResource NetServParams NetServResources where
         buildComponentR "netcli" allocate release
       where
         allocate = do
+            nsPeers' <- liftIO $ Set.fromList <$> mapM mkZTNodeId (Set.toList nsPeers)
+            nsOurAddress' <- liftIO $ mkZTNodeId nsOurAddress
+            nsInternalAddress' <-
+                liftIO $ maybe (pure Nothing) (fmap Just . mkZTNodeId) nsInternalAddress
```

Comment:
Isn't `maybe (... Nothing) (... Just ...)` a `fmap x`? Or, maybe whole expression can be written as `(fmap.fmap) mkZTNodeId nsInternalAddress`?